docs/reference/cluster_manifest.md

```diff
@@ -359,3 +364,25 @@ CPU and memory limits for the sidecar container.
 * **memory**
   memory limits for the sidecar container. Optional, overrides the
   `default_memory_limits` operator configuration parameter. Optional.
+
+## Connection pool
+
+Parameters are grouped under the `connectionPool` top-level key and specify
+configuration for connection pool. If this section is not empty, a connection
+pool will be created for a database even if `enableConnectionPool` is not
+present.
+
+* **number_of_instances**
```

Comment:
we have this convention to make manifest fields CamelCase and the config options snake_case. Any reason `dockerImage` isn't listed here?